README.md

```diff
@@ -1,20 +1,22 @@
+React is a fun and interactive front end web framework that is quickly taking over the world! We want to create a 4-week bootcamp about React for students who have some knowledge of JavaScript and are ready to pick up a new technology. Our focus will be on the functional components and React Hooks, as opposed to React class components.
 
 **Why Hooks instead of the traditional classes?**
 
-You only have one state object per component class in React which means there will be times where fields are placed together in the state object that has nothing to do with each other. This makes the code confusing. You need to bind 'this' for every event handler function. This is not intuitive and can be difficult for React newbies to understand.
+With only one state object per component class in React, unrelated fields may be placed together in the state object. This makes the code confusing and hard-to-read. You also need to bind `this` for every event handler function. This is not intuitive, and it can be difficult for React newbies to understand.
 
-Hooks were created to address the above concerns and several others. Hooks make React much cleaner, easier to read, and easier to write. The developers of React see Hooks as its future. They do not currently have plans to remove class support but within the Hooks FAQ they state "In the longer term, we expect Hooks to be the primary way people write React components." Hence, I believe Hooks are the proper way to teach React in 2020.
+Hooks were created to address the above concerns and several others. They make React much cleaner, easier to read, and easier to write. Although there are currently no plans to remove class support, it is stated in the Hooks FAQ: "In the longer term, we expect Hooks to be the primary way people write React components." Hence, I believe Hooks are the proper way to teach React in 2020. 
 
 **What will the students work on?**
 
-The students will create a Todo list, an Expense Calculator, and a Snippet Generator. 
+The students will create a Video Player, a To-Do List app, an Expense Calculator, and a Snippet Generator. 
 
-Many other React tutorials begin with a Todo list as it is a simple assignment that is able to show off many of features React offers. The Todo list will be constructing in class and will retirerate all of the concepts of JSX and React Hooks that the students have seen up to that point. Then they will complete an Expense Calculator to reinforce these principles.
+The Video Player will be an assignment where students, given starter code, will create a video player that can swap between four videos. It will be a good opportunity for students to understand how a child can pass information to its siblings.
```

Comment:
I added one or two sentences about the video player assignment here, since it wasn't mentioned in the "list of stuff you build". Let me know what you think.